src/Editor/ImGui/GuiHelpers.cpp

```diff
@@ -2,7 +2,7 @@
 
 namespace ImGui {
     void ShowHelpMarker(const char* description, const float sameLineOffset) {
-        if(sameLineOffset > 0.0f)
+        if (sameLineOffset > 0.0f)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') keyword 'if' not followed by a single space [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=vera%2B%2B%3AT008)